### DISCUSSION TITLE
Keymaps Set Detection

### DISCUSSION THREAD

**Discussion by m4xshen** [LazyVim/LazyVim#1029] [Q&A] [answered]
I want to execute some lua function after the default keymaps are set. I planned to use VimEnter autocmd but I found that keymaps are set after VimEnter. Is there any other way to achieve this?

**Comment by folke** [comment]
yes, just execute that code in your own `lua/config/keymaps.lua`
That is guaranteed to run after the global lazyvim keymaps.

Be aware that this is different for keymaps set by plugins. Refer to the plugin docs for more info about this.

**Reply by m4xshen** [reply]
Yeah I'm talking about keymaps set by plugins here. I found the problem is that when I open LazyVim directly, the plugin is loaded before the keymap. But when I open LazyVim with files, the plugin is loaded after the keymap. How can I make the plugin always loads after the keymap? (I'm installing the plugin with `event = "VeryLazy"`)

**Reply by folke** [reply, marked as answer]
https://www.lazyvim.org/plugins#%EF%B8%8F-adding--disabling-plugin-keymaps

**Reply by m4xshen** [reply]
Thanks
